### PR TITLE
only fire giraffe js on form

### PR DESCRIPTION
**Pull request by AWare**
don't run the giraffe js on the thank you page

### CODE REVIEW

File: frontend/assets/javascripts/src/modules/giraffe.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ define(['src/modules/form/validation/display','ajax'], function (display, ajax)
 
 
     function init() {
-        if (!document.querySelectorAll('.container-global--giraffe').length) {
+        if (!document.querySelectorAll('.container-global--giraffe .js-form').length) {
```
Comment (joelochlann) [inline frontend/assets/javascripts/src/modules/giraffe.js]:
```javascript
if (!document.querySelector(...)) {
    ...
}
```
would save a few precious chars :stuck_out_tongue_winking_eye:

**Comment by joelochlann** [comment]
:+1:

File: frontend/assets/javascripts/src/modules/giraffe.js (hunk 1 above)
Comment (joelochlann) [inline frontend/assets/javascripts/src/modules/giraffe.js]: and maybe we should just stick a more specific class on the giraffe contribute page

**Comment by AWare** [comment]
@joelochlann done

**Comment by joelochlann** [comment]
:+1: :+1:

**Pull request merged**